src/nvim/screen.c

```diff
@@ -1943,6 +1943,44 @@ static inline void get_line_number_str(win_T *wp, linenr_T lnum, char_u *buf, si
   snprintf((char *)buf, buf_len, fmt, number_width(wp), num);
 }
 
+static void set_line_and_cul_attrs(win_T *wp, linenr_T lnum, int *line_attr, int *cul_attr,
```

Comment:
This function name is not very clear about what it does. `apply_cursorline_highlight` may be a better name.

Reply:
I was hoping you would suggest something here. 😄